gatsby/src/templates/custom-page/custom-page.tsx

```diff
@@ -33,7 +33,10 @@ const CustomPage: React.FC<IProps> = ({ data, pageContext }) => {
         body={page.content.processed}
         title={page.title}
         langCode={pageContext.langCode}
-        breadcrumbItems={[{ title: I18n('home'), url: '/' }, page.title]}
+        breadcrumbItems={[
+          { title: I18n('home'), url: TRoute('/') },
```

Comment:
Nedala by se funkce TRoute používat jenom v té jedné komponentě schema? Ať nemusí být naimportovaná a použitá všude…

Reply:
Zkusím otestovat odpoledne, jestli nebude dělat bordel, ale teroreticky by to mělo stačit.

Reply:
Opraveno v novém commitu